coulomb-physical-constants/src/main/scala/coulomb/physicalconstants/package.scala

```diff
@@ -0,0 +1,191 @@
+    }
+
+    trait ElementaryCharge
+    implicit def defineElementaryCharge = {
+      // original value = 1.602176634×10−19 C
+      val denom = Rational(10).pow(19)
+      val num = Rational(1602176634L) / Rational(10).pow(9)
+      DerivedUnit[ElementaryCharge, Coulomb](coef = num / denom, name= "elementary-charge", abbv = "e")
+    }
+
+    trait HyperfineTransitionFrequencyOfCs133
+    implicit def defineHyperfineTransitionFrequencyOfCs133 = {
+      // original value = 9192631770 Hz
+      DerivedUnit[HyperfineTransitionFrequencyOfCs133, Hertz](coef = 9192631770L, name= "hyperfine-transition-frequency-of-cs-133", abbv = "∆Cs")
+    }
+
+    trait AvogadroConstant
+    implicit def defineAvogadrConstant = {
+      // original value = 6.02214076×1023 mol-1
+      val denom = Rational(10).pow(23)
+      val num = Rational(602214076L) / Rational(10).pow(8)
+      DerivedUnit[AvogadroConstant, Mole %^ -1](coef = num / denom, name= "avogadro-constant", abbv = "Nₐ")
+    }
+
+    trait BoltzmannConstant
+    implicit def defineBoltzmannConstant = {
+      // original value = 1.380649×10−23 J⋅K−1
+      val denom = Rational(10).pow(23)
+      val num = Rational(1380649L) / Rational(10).pow(6)
+      DerivedUnit[BoltzmannConstant, Joule %/ Kelvin](coef = num / denom, name= "boltzmann-constant", abbv = "k")
+    }
+
+    trait ConductanceQuantum
+    implicit def defineConductanceQuantum = {
+      // original value = 7.748091729...×10−5
+      val denom = Rational(10).pow(5)
+      val num = Rational(7748091729L) / Rational(10).pow(9)
+      DerivedUnit[ConductanceQuantum, Siemens](coef = num / denom, name= "conductance-quantum", abbv = "G₀")
+    }
+
+    trait JosephsonConstant
+    implicit def defineJosephsonConstant = {
+      // original value = 483597.8484...×109 Hz⋅V−1
+      //
+      val factor = Rational(10).pow(9)
+      val num = Rational(4835978484L) / Rational(10).pow(4)
+      DerivedUnit[JosephsonConstant, Hertz %/ Volt](coef = num * factor, name= "josephson-constant", abbv = "Kⱼ")
+    }
+
+    trait VonKlitzingConstant
+    implicit def defineVonKlitzingConstant = {
+      // original value = 25812.80745... Ω
+      val coef = Rational(2581280745L) / Rational(10).pow(5)
+      DerivedUnit[VonKlitzingConstant, Ohm](coef = coef, name= "von-klitzing-constant", abbv = "Rₖ")
+    }
+
+    trait MagneticFluxQuantum
+    implicit def defineMagneticFluxQuantum = {
+      // original value = 2.067833848...×10−15 Wb
+      val denom = Rational(10).pow(15)
+      val num = Rational(2067833848L) / Rational(10).pow(9)
+      DerivedUnit[MagneticFluxQuantum, Weber](coef = num / denom, name= "magnetic-flux-quantum", abbv = "Φ₀")
+    }
+
+  }
+
+  package infra {
+
+    trait PhysicalConstantQuantity[V, CU] {
+      type QU
+      val q: Quantity[V, QU]
+    }
+    object PhysicalConstantQuantity {
+      type Aux[T, V, CU] = PhysicalConstantQuantity[V, CU] {type QU = T}
+
+      implicit def summon[V, CU, U](implicit
```

Comment:
I was starting to like using `summon` but it's going to be a keyword in scala 3 so probably should avoid it

Reply:
Good call

Reply:
can you rename that to `summonConstant` or some such?

Reply:
Renamed